exp_build/task.js

```diff
@@ -282,6 +282,23 @@ class Task extends Unit {
           }
         }
         break;
+      
+      case ContentTypes.COMPONENT:
+        page_data.isComponent = true;
+        let css_module = [];
+        let js_module = [];
+        exp_info.codeassessment_js_modules.forEach(jsmodule => {
```

Comment:
Please remove the code-assessment specific code from here and build the component type in a generic manner

Reply:
done